io.catenax.message_header/1.0.0/MessageHeaderAspect.ttl

```diff
@@ -0,0 +1,126 @@
+   samm:description """Information about the context the message should be considered in.
+
+Default: For Use Cases which need to validate against the Semantic Hub automatically.
+
+Fallback: For Use Cases which do not use automatic validation.
+
+In both cases: Versioning only refers to major versions.
+
+String pattern of the value follows: <domain>-<subdomain>-<object>-<major version>"""@en ;
+   samm:characteristic :ContextCharacteristic ;
+   samm:exampleValue "urn:samm:io.catenax.<trace-QM-Alert>:1.x.x" .
+
+:sentDateTime a samm:Property ;
+   samm:preferredName "Sent Date Time"@en ;
+   samm:description """Time zone aware timestamp holding the date and the time the message was sent by the sending party
+
+The value MUST be formatted according to the ISO 8601 standard"""@en ;
+   samm:characteristic :Timestamp ;
+   samm:exampleValue "2023-06-19T21:24:00+07:00"^^xsd:dateTimeStamp .
+
+:senderBpn a samm:Property ;
+   samm:preferredName "Sender BPN"@en ;
+   samm:description """The Business Partner Number of the sending party.
+
+The value MUST be a valid BPN. Applicable constraints are defined in the corresponding standard"""@en ;
+   samm:characteristic :BpnCharacteristic ;
+   samm:exampleValue "BPNL7588787849VQ" .
+
+:receiverBpn a samm:Property ;
+   samm:preferredName "Receiver BPN"@en ;
+   samm:description """The Business Partner Number of the receiving party.
+
+The value MUST be a valid BPN. Applicable constraints are defined in the corresponding standard."""@en ;
+   samm:characteristic :BpnCharacteristic ;
+   samm:exampleValue "BPNL6666787765VQ" .
+
+:expectedResponseBy a samm:Property ;
+   samm:preferredName "Expected Response By"@en ;
+   samm:description """Time zone aware timestamp holding the date and time by which the sending party expects a certain type of response from the receiving party.
+
+The meaning and interpretation of the fields's value are context-bound and MUST therefore be defined by any business domain or platform capability making use of it.
+
+The value MUST be formatted according to the ISO 8601 standard"""@en ;
+   samm:characteristic :Timestamp ;
+   samm:exampleValue "2023-06-19T21:24:00+07:00"^^xsd:dateTimeStamp .
+
+:relatedMessageId a samm:Property ;
+   samm:preferredName "Related Message ID"@en ;
+   samm:description "Unique ID identifying a message somehow related to the current one"@en ;
+   samm:characteristic :UuidCharacteristic ;
+   samm:exampleValue "d9452f24-3bf3-4134-b3eb-68858f1b2362" .
+
+:version a samm:Property ;
+   samm:preferredName "Version"@en ;
+   samm:description """The version of the standard defining the semantics and the structure of the message's header.
+
+The version number should follow the Semantic Versioning 2.0.0 specification"""@en ;
+   samm:characteristic :VersionCharacteristic ;
+   samm:exampleValue "urn:samm:io.catenax.message_header:1.0.0" .
+
+:UuidCharacteristic a samm:Characteristic ;
+   samm:preferredName "UUID Characteristic"@en ;
```

Comment:
add regex for uuid, check 
https://github.com/eclipse-tractusx/sldt-semantic-models/blob/cd4a327ab47b2891cbf071845b9d45c1a3fefa78/io.catenax.pcf/4.0.1/Pcf.ttl#L198

https://github.com/eclipse-tractusx/sldt-semantic-models/blob/cd4a327ab47b2891cbf071845b9d45c1a3fefa78/io.catenax.pcf/4.0.1/Pcf.ttl#L243

Reply:
trait has been added